src/test/java/Web/TestCases/NewDigitalId/ExpectedElementsArePresentAtStepIdType.java

```diff
@@ -1,65 +1,49 @@
 package Web.TestCases.NewDigitalId;
-
-import Web.SiteTree.Services.InternalServices.DigitalIdRequests.NewId.NewDigitalIdRequest;
-import Web.SiteTree.Services.InternalServices.DigitalIdRequests.NewId.Steper.IdType.IdTypeStep;
-import Web.SiteTree.Services.InternalServices.InternalServices;
+import Pages.App.*;
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.NewDigitalIdRequestPage;
+import Pages.Services.InternalServices.DigitalIdRequests.NewId.Steper.IdType.IdTypeStepPage;
+import Pages.Services.InternalServices.InternalServicesPage;
+import Pages.Services.ServicesPage;
 import Web.TestCases.Login.NeomUserCanBeSuccessfullyLoggedIn;
-import Web.SiteTree.Services.*;
-import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.ElementsCollection;
-import org.testng.annotations.Test;
-
-import java.lang.reflect.Array;
-import java.time.Duration;
+import org.junit.jupiter.api.BeforeEach;
+import org.testng.annotations.*;
 import java.util.Arrays;
 import java.util.List;
-
 import static com.codeborne.selenide.Condition.*;
 import static com.codeborne.selenide.Selenide.*;
 
 public class ExpectedElementsArePresentAtStepIdType extends NeomUserCanBeSuccessfullyLoggedIn {
-    @Test
-    public void expectedElementsArePresentAtStepIdType(){
+    private App app;
+    @BeforeMethod
+    public void setUp(){
```

Comment:
this is common setup, so you have to create a base test class, keep this common setup there
your test class should extend base test class and inherit it